service/db/src/main/java/gov/va/vro/service/db/SaveToDbServiceImpl.java

```diff
@@ -30,6 +39,36 @@ public Claim insertClaim(Claim claim) {
     return claim;
   }
 
+  @Override
+  public void insertAssessmentResult(
+      UUID claimId, AbdEvidenceWithSummary evidenceResponse, String diagnosticCode) {
+    ClaimEntity errorClaim = new ClaimEntity();
+    errorClaim.setIncomingStatus("ERROR");
+    ClaimEntity claimEntity = claimRepository.findById(claimId).orElse(errorClaim);
+    if (claimEntity.getIncomingStatus().equals("ERROR")) {
+      log.warn("Could not match Claim ID in insertAssessmentResult, exiting.");
+      return;
+    }
+    Map summary = evidenceResponse.getEvidenceSummary();
+    AssessmentResultEntity assessmentResultEntity = new AssessmentResultEntity();
```

Comment:
Do you not want to check summary for null or empty?

Reply:
@au2 We check for null in AssessmentResultProcessor before sending it over here, but I can do a second check.